plugins/cliconf/vyos.py

```diff
@@ -269,12 +272,14 @@ def get_diff(
                 updates.append(line)
 
             elif item.startswith("delete"):
+
                 if not running_commands:
                     updates.append(line)
                 else:
                     item = re.sub(r"delete", "set", item)
+
                     for entry in running_commands:
-                        if entry.startswith(item) and line not in visited:
+                        if entry.startswith(item + " ") and line not in visited:
```

Comment:
Is this going to get an exact match as well as matching items below it in the hierarchy?  Seems like it would fail to match the exact item.